### PR TITLE
add logging infrastructure

### PR DESCRIPTION
**Pull request by romanlutz**

### CODE REVIEW

File: fairlearn/__init__.py [hunk 1]
```diff
@@ -1,4 +1,17 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
 __name__ = "fairlearn"
 __version__ = "0.3.0-alpha"
 
 _KW_SENSITIVE_FEATURES = "sensitive_features"
+
+
+# Setup logging infrustructure
```
Comment (riedgar-ms) [inline fairlearn/__init__.py]: infrastructure

Comment (romanlutz) [reply fairlearn/__init__.py]: Ah! Well spotted!

File: fairlearn/__init__.py [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+import logging  # noqa: E402
+import os  # noqa: E402
```
Comment (riedgar-ms) [inline fairlearn/__init__.py]: Why the flake8 suppression?

Comment (romanlutz) [reply fairlearn/__init__.py]: E402 module level import not at top of file

I don't want to put it there because this file is mainly for the version

**Comment by romanlutz** [comment]
closing since rebasing for DCO is ridiculously hard. Starting a new PR

**Pull request closed without merging**